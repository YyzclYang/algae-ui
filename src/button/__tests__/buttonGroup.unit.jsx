import React from 'react';
import * as renderer from 'react-test-renderer';
import Button, { ButtonGroup } from '../index';

describe('ButtonGroup', () => {
  it('渲染 ButtonGroup', () => {
    const component = renderer
      .create(
        <ButtonGroup>
          <Button>Button1</Button>
          <Button>Button2</Button>
          <Button>Button3</Button>
        </ButtonGroup>
      )
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
