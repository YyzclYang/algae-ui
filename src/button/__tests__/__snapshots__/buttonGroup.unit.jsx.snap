// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonGroup 渲染 ButtonGroup 1`] = `
<div
  className="algae-ui-button-group"
>
  <button
    className="algae-ui-button algae-ui-button-default"
    onClick={[Function]}
  >
    Button1
  </button>
  <button
    className="algae-ui-button algae-ui-button-default"
    onClick={[Function]}
  >
    Button2
  </button>
  <button
    className="algae-ui-button algae-ui-button-default"
    onClick={[Function]}
  >
    Button3
  </button>
</div>
`;
